app_dart/lib/src/request_handlers/update_existing_flaky_issues.dart

```diff
@@ -106,11 +106,10 @@ class UpdateExistingFlakyIssue extends ApiRequestHandler<Body> {
   }) async {
     final Map<String, bool> builderFlakyMap = <String, bool>{};
     final Map<String, bool> ignoreFlakyMap = <String, bool>{};
-    for (pb.Target target in schedulerConfig.targets) {
-      builderFlakyMap[target.name] = target.bringup;
-      if (target.properties.containsKey(kCiYamlTargetIgnoreFlakiness) &&
-          target.properties[kCiYamlTargetIgnoreFlakiness] == 'true') {
-        ignoreFlakyMap[target.name] = true;
+    for (Target target in schedulerConfig.targets as List<Target>) {
```

Comment:
I'm getting this failure when running tests:

```
type 'PbList<Target>' is not a subtype of type 'List<Target>' in type cast where
  Target is from package:cocoon_service/src/model/proto/internal/scheduler.pb.dart
  Target is from package:cocoon_service/src/model/ci_yaml/target.dart

package:cocoon_service/src/request_handlers/update_existing_flaky_issues.dart 109:51  UpdateExistingFlakyIssue._updateExistingFlakyIssue
package:cocoon_service/src/request_handlers/update_existing_flaky_issues.dart 52:11   UpdateExistingFlakyIssue.get
```

Reply:
Why are we using pb.Target instead of the CiYaml.postsubmitTargets